{
  "Header": {
    "Original": "CVE-2017-11340 (There is a Segmentation fault in the XmpParser::terminate() function i ...)",
    "ID": "CVE-2017-11340",
    "Description": "(There is a Segmentation fault in the XmpParser::terminate() function i ...)"
  },
  "Annotations": [
    {
      "Original": "- exiv2 \u003cnot-affected\u003e (Vulnerable code introduced after 0.25; only affected experimental; bug #868578)",
      "Type": "package",
      "Package": "exiv2",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced after 0.25; only affected experimental; bug #868578"
    },
    {
      "Original": "NOTE: https://github.com/Exiv2/exiv2/issues/53",
      "Type": "NOTE",
      "Description": "https://github.com/Exiv2/exiv2/issues/53"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1470950",
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1470950"
    },
    {
      "Original": "NOTE: Not reproducible in wheezy/jessie/stretch, I get \"The file contains data of an unknown image type\".",
      "Type": "NOTE",
      "Description": "Not reproducible in wheezy/jessie/stretch, I get \"The file contains data of an unknown image type\"."
    },
    {
      "Original": "NOTE: Reproducible with 0.26-1 (experimental) although I get another error \"free(): invalid next size (fast)\".",
      "Type": "NOTE",
      "Description": "Reproducible with 0.26-1 (experimental) although I get another error \"free(): invalid next size (fast)\"."
    }
  ]
}